app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,13 @@
+from players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str, favourite_dish: str, skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def get_rating(self) -> int:
+        return self._skill_level
+
+    def player_info(self) -> str:
+        return f"Dwarf Blacksmith: {self.nickname}"
```

Comment:
Code Style: Use one style of quotes in your code. Double quotes are preferable. Consider changing the single quotes to double quotes in the return statement.